rnote-compose/src/shapes/arrow.rs

```diff
@@ -143,7 +143,7 @@ impl Arrow {
     pub fn compute_rline(&self, stroke_width: Option<f64>) -> na::Vector2<f64> {
         let vec_b =
             self.compute_stem_direction_vector() * Self::compute_tip_lines_length(stroke_width);
-        let rotation_matrix = self.get_rotation_matrix().transpose();
+        let rotation_matrix = Rotation2::new(Self::TIP_LINES_STEM_ACUTE_ANGLE);
```

Comment:
the angle needs to be negative for rline

Reply:
Aye! Fixed with pushed commit.